pco/src/data_types/mod.rs

```diff
@@ -108,7 +109,12 @@ pub trait Latent:
   /// impossible.
   fn from_u64(x: u64) -> Self;
 
+  fn is_zero(self) -> bool {
+    self == Self::ZERO
+  }
+
   fn leading_zeros(self) -> Bitlen;
+  fn trailing_zeros(self) -> Bitlen;
```

Comment:
We've added two trailing zeros functions (here and in Latent), but I think we need just this one?